src/mobot_client/models/__init__.py

```diff
@@ -1,65 +1,64 @@
-# Copyright (c) 2021 MobileCoin. All rights reserved.
-import enum
+#  Copyright (c) 2021 MobileCoin. All rights reserved.
 
-from django.db import models
+from decimal import Decimal
 
+from django.db import models
+import mobilecoin as mc
 
-class SessionState(models.IntegerChoices):
-    OUT_OF_MOB = -2
-    CANCELLED = -1
-    READY_TO_RECEIVE_INITIAL = 0
-    WAITING_FOR_BONUS_TRANSACTION = 1
-    ALLOW_CONTACT_REQUESTED = 2
-    COMPLETED = 3
+from mobot_client.models.states import SessionState
 
 
 class Store(models.Model):
     name = models.TextField()
-    phone_number = models.TextField()
+    phone_number = models.TextField(db_index=True)
```

Comment:
Nit: This shouldn't be a text field. There's a nice Django library for handling phone numbers (https://github.com/stefanfoulis/django-phonenumber-field).

Reply:
So, like before, what you're looking at is a minimal branch. I'm working up to replacing these bit by bit. The phonenumbers field will go in when we're more in the clear, and it exists in the refactor branches.

Reply:
Ended up just moving to Phone numbers. It's been an ambitious weekend.